scripts/benchmarks/benches/velocity.py

```diff
@@ -29,7 +28,11 @@ def download_deps(self):
 
     def setup(self):
         self.download_deps()
-
+        self.benchmark_bin = os.path.join(self.directory, self.bench_name, self.bin_name)
+        
+        if options.no_git:
```

Comment:
I'm not the biggest fan of the way this option is implemented. Would it be possible to just do this inside of `git_clone` function?
That way, it will simply skip the fetch, and the cmake/make should not need to do anything.

Reply:
The option prevents from any directory removal, build etc. Can be removed in a version we consider stable, as cluster node is not destination environment for the script